src/gcp_scanner/arguments.py

```diff
@@ -143,5 +147,11 @@ def arg_parser():
         'Please select at least one option to begin scan\
  -k/--sa-key-path,-g/--gcloud-profile-path, -m, -rt, -at'
     )
+  if not os.path.isdir(args.output):
+    logging.error(
+      '\"%s\" doesn\'t exists. Please enter a valid directory path.', 
```

Comment:
Oh, actually small nit here. Please fix `exists`, It should be `exist`

Reply:
Done :+1: